package.json

```diff
@@ -11,9 +11,9 @@
     "build:js": "rollup -c config/rollup.config.js",
     "build:scss": "sh ./scripts/cp-scss.sh",
     "build-package": "yarn build:js && yarn build:css && yarn build:scss",
-    "build-gitbook": "gitbook build guide guide/_book",
-    "build-styleguide": "styleguidist build --config config/styleguide.config.js",
-    "build:docs-staging": "yarn build-styleguide && gitbook build guide build/staging",
+    "build-docs:gitbook": "gitbook build guide",
```

Comment:
Out of curiosity, why do we have two namespace conventions: dashes and colons?

Reply:
"dash" means a multi-word task such as "build documentation"
"colon" means a sub task such as "build documentation:gitbook" and "build documentation:styleguide"

Normally, the "main" task calls all the subtasks.

That's what Laura and I discussed when we made this change.

Thoughts?

Reply:
Sort of related to this topic, we have `yarn dev` to run styleguidist and `yarn gitbook` to run gitbook.

What are your thoughts on updating `yarn dev` to `yarn styleguidist` :)